src/main/java/be/seeseemelk/mockbukkit/entity/PlayerMock.java

```diff
@@ -1260,50 +1253,44 @@ public void setBedSpawnLocation(Location location, boolean force)
 	@Override
 	public boolean getAllowFlight()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return canFly;
 	}
-	
+	boolean canFly;
+	boolean isFlying;
 	@Override
 	public void setAllowFlight(boolean flight)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		canFly = flight;
 	}
 	
 	@Override
 	public void hidePlayer(Player player)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		hiddenPlayers.put(this,player);
 	}
 	
 	@Override
 	public void showPlayer(Player player)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		hiddenPlayers.remove(this,player);
 	}
-	
+	HashMap<Player,Player> hiddenPlayers = new HashMap<>();
 	@Override
 	public boolean canSee(Player player)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return hiddenPlayers.containsKey(this) && hiddenPlayers.containsValue(player);
 	}
 	
 	@Override
 	public boolean isFlying()
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		return this.isFlying;
 	}
 	
 	@Override
 	public void setFlying(boolean value)
 	{
-		// TODO Auto-generated method stub
-		throw new UnimplementedOperationException();
+		this.isFlying = true;
 	}
```

Comment:
Argument is ignored :face_with_thermometer:

Reply:
bruh im dumb lol